src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -167,10 +169,12 @@ exprt value_set_dereferencet::dereference(
     compare_against_pointer = fresh_binder.symbol_expr();
   }
 
-  std::list<valuet> values =
-    make_range(retained_values).map([&](const exprt &value) {
-      return build_reference_to(value, compare_against_pointer, ns);
-    });
+  auto values =
+    make_range(retained_values)
+      .map([&](const exprt &value) {
+        return build_reference_to(value, compare_against_pointer, ns);
+      })
+      .collect<std::deque<valuet>>();
```

Comment:
This one is the one exception here, this doesn’t make it easier to follow it just bothers me to see a `std::list` if we don’t need the flexibility.